src/main/java/com/alipay/oceanbase/rpc/table/ObHBaseParams.java

```diff
@@ -28,6 +28,7 @@ public class ObHBaseParams extends ObKVParamsBase {
     boolean                  allowPartialResults        = true;  // whether allow partial row return or not
     boolean                  isCacheBlock               = false; // whether enable server block cache and row cache or not
     boolean                  checkExistenceOnly         = false; // check the existence only
+    String                   hbaseVersion               = "1.3.6";
```

Comment:
maybe we can use org.apache.hadoop.hbase.util.VersionInfo to get hbaseVersion?

Reply:
obkv table client do not rely on hbase, unable to get correct hbase version here